Mathlib/LinearAlgebra/Span/Basic.lean

```diff
@@ -80,6 +80,57 @@ theorem span_preimage_le (f : F) (s : Set M₂) :
 
 alias _root_.LinearMap.span_preimage_le := Submodule.span_preimage_le
 
+section
+
+variable {N : Type*} [AddCommMonoid N] [Module R N]
+
+lemma linearMap_eq_iff_of_eq_span {V : Submodule R M} (f g : V →ₗ[R] N)
```

Comment:
Should this be called `ext_iff_of_eq_span` or something?

Reply:
I do not think so.

Reply:
> Should this be called `LinearMap.submodule_ext_iff_of_eq_span` or something?

I like this name too. It enables dot notation, and the iff version of extensionality lemmas are usually called `ext_iff`, e.g. [MvPolynomial.funext_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/MvPolynomial/Funext.html#MvPolynomial.funext_iff), [DFunLike.ext_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/FunLike/Basic.html#DFunLike.ext_iff)